### PR TITLE
Improve SDK Documentation

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

File: docs/sdk/transaction.md [hunk 1]
```diff
@@ -130,12 +189,19 @@ if (details.unsigned.scriptOpt !== undefined && !details.scriptExecutionOk) {
...
 ## Decode Unsigned Transaction
 
+The Alephium full node also provides an API to decode unsigned
+transaction from its serialized format:
+
 ```typescript
 const nodeProvider = new NodeProvider('http://127.0.0.1:22973')
-const decodedTx = await nodeProvider.transactions.postTransactionsDecodeUnsignedTx({ unsignedTx })
+const decodedUnsignedTx = await nodeProvider.transactions.postTransactionsDecodeUnsignedTx({ unsignedTx })
+console.log('decoded unsigned transaction, decodedUnsignedTx)
```
Comment (Lbqds) [inline docs/sdk/transaction.md]:
```suggestion
console.log('decoded unsigned transaction ', decodedUnsignedTx)
```

File: docs/sdk/web3-react.md [hunk 2]
```diff
@@ -4,71 +4,123 @@ title: Web3 React
...
-
-## Hooks
-
-### useConnect
+For even more customization, we can use the `useConnect` hook, which is the
+used by `AlephiumConnectButton` under the hood:
 
 ```typescript
-import { useConnect } from '@alephium/web3-react'
+import { useConnect } from 'alephium-web3'
```
Comment (Lbqds) [inline docs/sdk/web3-react.md:90]: Should it be `@alephium/web3-react`?

**Review by Lbqds — APPROVED** [review]
LGTM 👍

File: docs/sdk/web3-react.md (hunk 2 above)
Comment (h0ngcha0) [reply docs/sdk/web3-react.md:90]: Great catch :+1:

**Pull request merged**